File: Loops/FizzBuzz.py
#Evelyn Chennault, FizzBuzz Python

while x:
    x = int(input("Type in a number between 1 and 100:\n"))
    if x % 3 and x % 5 == 0:
        print("FizzBuzz")
    elif x % 3 == 0:
        print("Fizz")
    elif x % 5 == 0:
        print("Buzz")
    else: 
        print(x)
    x += 1

File: Loops/FizzBuzz.c
//Evelyn Chennault, FizzBuzz C
#include <stdio.h>
#include <string.h>
#include <math.h>

int main(void){
    int num;
    for(num){
        if(num % 3){
            printf("Fizz");
        }else if(num % 5){
            printf("Buzz");
        }else if (num % 3 && num % 5){
            printf("FizzBuzz");
        }else{
            printf(num);
        }
    return 0;
}